src/ansys/dpf/composites/unit_system.py

```diff
@@ -0,0 +1,39 @@
+"""Unit system helper."""
+__all__ = ("get_unit_system",)
+
+from typing import Optional, Union
+
+from ansys.dpf.core import DataSources, Operator, ResultInfo, UnitSystem
+
+
+def get_unit_system(
+    rst_data_source: DataSources, default_unit_system: Optional[UnitSystem] = None
+) -> Union[UnitSystem, ResultInfo]:
```

Comment:
We have to use a Union because it is not possible to get the UnitSystem from ResultInfo in python.